test/karma-conf.js

```diff
@@ -349,6 +349,10 @@ module.exports = function (config) {
         port: 9876,  // karma web server port
         colors: true,
         logLevel: config.LOG_INFO,
+        browserConsoleLogOptions: {
+            path: `${process.cwd()}/test/console.log`,
+            terminal: false
```

Comment:
Might be good to set this based on `process.env.CI` to keep the output there. Alternatively we could `cat` the output files after [running tests in workflows](https://github.com/highcharts/highcharts/blob/master/.github/workflows/highcharts-headless.yml#L108) (which would be a good separation actually)

Reply:
Okay, I'm happy with either as long as it doesn't clutter up the terminal while debugging. You fix?